contracts/liquidity_hub/pool-network/incentive/src/contract.rs

```diff
@@ -172,7 +172,7 @@ pub fn migrate(mut deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Respons
         });
     }
 
-    if storage_version < Version::parse("1.0.4")? {
+    if storage_version < Version::parse("1.0.6")? {
```

Comment:
Might be good to keep a record somewhere of what changes between versions just for your own reference

Reply:
Yes we should probably add a changelog file to the repo, good point.

Reply:
Yes we should probably add a changelog file to the repo, good point.